src/include/target/Frsky_TX_R9M.h

```diff
@@ -42,9 +42,9 @@
 // Power Output
 #define POWER_OUTPUT_DAC                0b0001100
 #define MinPower                        PWR_10mW
-#define HighPower                       PWR_250mW
-#define MaxPower                        PWR_1000mW
+#define HighPower                       PWR_2000mW
+#define MaxPower                        PWR_2000mW
 #if !defined(POWER_OUTPUT_VALUES)
-    #define POWER_OUTPUT_VALUES         {720,875,1000,1140,1390,1730,2100,2600}
-    #define POWER_OUTPUT_VALUES_868     {650,860,1000,1160,1420,1730,2100,2600}
+    #define POWER_OUTPUT_VALUES         {720,875,1000,1140,1390,1730,2300,3300}
+    #define POWER_OUTPUT_VALUES_868     {650,860,1000,1160,1420,1730,2300,3300}
```

Comment:
This looks to be driving the V_APC beyond its max of 2.8V and not much is achieved going beyond the typical.

![image](https://user-images.githubusercontent.com/14170229/218876600-7036cdc7-3f1c-49ae-a7fe-21ff51a41b84.png)

![image](https://user-images.githubusercontent.com/14170229/218877123-51977797-5b77-41c6-83f9-eb5c56591dbd.png)

Reply:
2900, aka 2.9 volts is where the peak dBm was achieved with our tests across many units. But we also saw no adverse effects when running at 3.3 volts, as there is overcurrent protection. As long as this is 2900 that should be fine, as 3.3 isn't really necessary either way.